e2e/playwright.config.ts

```diff
@@ -51,12 +51,17 @@ const config: PlaywrightTestConfig = {
     baseURL: 'http://localhost:3000',
   },
 
+  expect: {
+    // Tests are really show on CI, increase the timeout
```

Comment:
```suggestion
    // Tests are really slow on CI, increase the timeout
```

Please check for other places too which might have this typo